golded3/gcarea.cpp

```diff
@@ -184,7 +184,7 @@ void AreaList::AddNewArea(AreaCfg* aa)
   }
 
   Desc desc;
-  char buf[256];
+  char buf[2560];
```

Comment:
Is it possible to move this value into a shared constant and use this constant everywhere instead of multiple hardcoded values?

Reply:
I think it is possible but not everywhere. Only where it is the size of a screen output buffer. Please do it.